src/core/zowe/core_for_zowe_sdk/validators.py

```diff
@@ -31,10 +33,22 @@ def validate_config_json(path_config_json: str, path_schema_json: str):
         Provides details if config.json doesn't match schema.json, otherwise it returns None.
     """
 
+    # checks if the path_schema_json point to an internet URI and download the schema using the URI
+    if path_schema_json[:8] in ("https://", "http://"):
+        response = requests.get(path_schema_json)
+        schema_data = response.content.decode("utf-8")
+        schema_json = commentjson.load(schema_data)
+    
+    # checks if the path_schema_json is a file
+    elif os.path.isfile(path_schema_json):
+        with open(path_schema_json) as file:
+            schema_json = commentjson.load(file)
+        
+    # if there is no path_schema_json it will return None
+    else: 
+        return None
+    
     with open(path_config_json) as file:
         config_json = commentjson.load(file)
```

Comment:
Currently we are loading the config file with `commentjson` twice. We could increase performance by passing the loaded JSON object directly into `validate_config_json`. Perhaps something like this:
```python
def validate_config_json(path_config_json: Union[str, dict], path_schema_json: str):
    if isinstanceof(path_config_json, str):
        # load config from file name
    else:
        # use config object passed in
```